### DISCUSSION TITLE
How to stop LRV recording format??

### DISCUSSION THREAD

**Discussion by jamalionline333** [gopro/labs#632] [General]
I really dont need LRV format to save while recording. I am using Gopro Black7. As both formats are not linked each other so if we need to delete one file so its hard to find the same file to be deleted. For uploading we could do downgrade at our own side. please inform to stop it.

**Comment by dnewman-gpsw** [comment]
LRV files are used by the mobile app.  There is an option for later camera models to remove them (although not HERO12 where they are simply hidden.)  Sorry, these changes will not be available on HERO7 as they are no scheduled firmware releases for that camera or any of GP1 cameras.
